packages/icon/README.md

```diff
@@ -149,7 +149,7 @@ A `LGGlyph.Component`
 
 (For DS engineers)
 
-- After uploading a new SVG file to `src/glyphs`, replace all `fill` property values with `#000`.
+- After uploading a new SVG file to `src/glyphs`, replace all `fill` property values for strokes with `#000`.
```

Comment:
Is this true? Shouldn't it still be "replace all `fill` values...", but ensure that the root `svg` element does not have a full?